_data/nav.yml

```diff
@@ -309,6 +309,7 @@
     - url: "#image_class_list"
     - url: "#image_description"
     - url: "#image_dimensions"
+    - url: "#images_file_types"
```

Comment:
Bit of a nitpick but should this be lower so that we can keep the almost alphabetical order?